oaknational/Oak-Web-Application
# LESQ-517: Add specialist to the hamburger menu

## Description

Music year: 2008 - https://www.youtube.com/watch?v=Ic5vxw3eijY

- Adds internal link to Specialist pages to the the hamburger menu

## Issue(s)

Fixes #LESQ-517

## How to test

1. Go to https://deploy-preview-2325--oak-web-application.netlify.thenational.academy
2. Click on click on menu 
3. You should see Specialist on the menu options

## Screenshots

How it used to look (delete if n/a):
<img width="716" alt="Screenshot 2024-03-26 at 08 58 10" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/d1200b04-e0d5-4de0-945f-b670646e7f05">

How it should now look:
<img width="716" alt="Screenshot 2024-03-26 at 08 57 13" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/957cb30d-9174-424d-9f73-b3699526a07c">
<img width="1359" alt="Screenshot 2024-03-26 at 16 07 32" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/6dd68b07-f49d-4f05-809e-533526e28415">
